internal/ledgerutil/compare.go

```diff
@@ -23,9 +23,15 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	// AllDivsByKey - Filename for the json output that contains all divergences ordered by key
+	AllDivsByKey = "all_divergences_by_key.json"
```

Comment:
```suggestion
	allDivsByKey = "all_divergences_by_key.json"
```

Reply:
actually we could call them "differences" rather than "divergences"... I think that is a little easier to understand and say and shorten "diff" (unless you and Manish already landed on "divergences").